src/component/LayerCarousel/LayerCarousel.tsx

```diff
@@ -273,28 +287,32 @@ export default class LayerCarousel extends React.Component<LayerCarouselProps, L
 
     const extent = map.getView().calculateExtent();
     const mapProjection = map.getView().getProjection().getCode();
-    return this.props.layers.map((layer: any) => {
+    return this.props.layers.map((layer) => {
       const staticImageUrl = layer.get('staticImageUrl');
       const previewImageRequestUrl = layer.get('previewImageRequestUrl');
       let layerFt, requestUrl;
       if (!staticImageUrl) {
         if (layer.get('type') === 'WMTS' && previewImageRequestUrl) {
           requestUrl = previewImageRequestUrl;
-          layerFt = layer.getSource().getLayer();
+          layerFt = (layer.getSource() as OlSourceWMTS).getLayer();
```

Comment:
What is the purpose of `layer.getSource().getLayer()`?

Reply:
It returns the name of the WMTS layer, see [here](https://openlayers.org/en/latest/apidoc/module-ol_source_WMTS-WMTS.html#getLayer).